persist/storer.go

```diff
@@ -9,6 +9,8 @@ import (
 	"time"
 
 	"github.com/newrelic/infra-integrations-sdk/log"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
```

Comment:
Does `unix` package means that we will not give support to windows users?

Reply:
Good point. If there is no Go standard way to handle file permissions, we should consider creating a `storer_windows.go` in the near future.

Reply:
👍 nice catch, removed